apps/web/app/api/mutations/admin/useBetaCreateFile.ts

```diff
@@ -23,7 +25,7 @@ export function useBetaCreateFileItem() {
     onSuccess: () => {
       toast({
         variant: "default",
-        description: "File item created successfully",
+        description: t("adminCourseView.curriculum.lesson.toast.fileItemCreatedSuccessfully"),
```

Comment:
```suggestion
        description: t("adminCourseView.curriculum.lesson.toast.fileLessonCreatedSuccessfully"),
```
And proposes to improve the message. We want to gradually throw out the old nomenclature